ddtrace/contrib/kafka/patch.py

```diff
@@ -203,3 +232,14 @@ def traced_commit(func, instance, args, kwargs):
     core.dispatch("kafka.commit.start", [instance, args, kwargs])
 
     return func(*args, **kwargs)
+
+
+def serialize_key(instance, topic, key, headers):
+    ctx = _SerializationContext(topic, _MessageField.KEY, headers)
```

Comment:
We should check if `_SerializationContext` is None here. This code might be refactored/reused and this could introduce bugs